README.md

```diff
@@ -1,17 +1,28 @@
 # ensembl-genomio
 Pipelines to turn basic genomic data into Ensembl cores and back
 
-A set of ehive pipelines:
+This is a mulitlanguage (Perl, Python) repo providing eHive pipelines
+and various scripts (see below) to prepare genomic data and load it as [Ensembl core database](http://www.ensembl.org/info/docs/api/core/index.html) or to dump such core databases as file bundles.
 
-* Genome dumper: dumps flat files from an Ensembl core database.
-* Genome loader: creates an Ensembl core database from a set of flat files.
+The files composing data bundle are of various formats (fasta, gff3, json) and should follow the corresponding [specification](docs/BRC4_genome_loader.md#input).
```

Comment:
"The files composing a genomic data bundle are of various formats (fasta,...."

Suggestion to alter this sentence  on line 7